integration-test/src/test/java/com/datastrato/gravitino/integration/test/spark/hive/SparkHiveCatalogIT.java

```diff
@@ -201,6 +201,23 @@ void testHiveFormatWithStoredAs() {
     checkParquetFile(tableInfo);
   }
 
+  @Test
+  void testHiveFormatWithExternalTable() {
```

Comment:
for external table, one of feature is the data retains after dropping table, could you add related test?
1. set table location explicitly when creating table.
2. add `dropTable` after `checkTabelReadWrite`
3. check if data file still exists in the table location, add `checkDataFileExists`   like `checkDirExists`

Reply:
Sure! I am glad to do this. :)